subjects/mobile-dev/favorite-images/audit/README.md

```diff
@@ -1,17 +1,29 @@
 #### Functional
 
-> In order to run and hot reload app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app)
+> In order to run and hot reload the app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app)
+
+##### Create a new empty personal gallery.
 
 ###### Does the app show "No images selected" message when no images are loaded?
 
-###### Does the app have a select image button in the appbar?
+###### Does the app have a `select image` button in the `appbar`?
+
+##### Tap on the select image button.
 
-###### Tap on the select image button. Do you have an option to choose an image from gallery and take a picture?
+###### Do you have an option to `choose an image` from gallery and `take a picture`?
 
-###### Does the camera work both in iOS and Android?
+###### Does the camera work both on `iOS` and `Android`?
 
 ###### Does the app load chosen image?
 
-###### Tap on the image. Does it become full-size?
+###### Does the image opens in `full-size` once taped?
+
+###### Can you `zoom in and out` the image?
```

Comment:
Here `zoom in and out` doesn't seems to be a command, we could avoid to put it in brackets or if you want to keep this notation maybe we could have:
`zoom in` and `zoom out`

Reply:
done